cmd/mic/main.go

```diff
@@ -103,7 +103,10 @@ func main() {
 	config.Burst = int(clientQPS)
 	glog.Infof("Client QPS set to: %v. Burst to: %v", config.QPS, config.Burst)
 
-	immutableUserMSIsList := strings.Split(immutableUserMSIs, ",")
+	var immutableUserMSIsList []string
+	if immutableUserMSIs != "" {
```

Comment:
Can we please rename this to ImmutableUserMSIsParam or rename the ImmutableUserMSIs in the client to ImmutableUserMSIsMap ?

Reply:
It is already `ImmutableUserMSIsMap` in the client.

Reply:
Done!